### ISSUE TITLE
Accessibility: Group label of radio button groups not announced by screen readers

### ISSUE THREAD

**Issue by MarcoZehe** [microsoft/terminal#9567]
### Windows Terminal version (or Windows build number)

Windows Terminal Preview Version: 1.7.572.0

### Other Software

Narrator, NVDA, JAWS

### Steps to reproduce

1. Turn on Narrator.
2. Open Settings.
3. Tab to the Start Mode group of radio buttons.

### Expected Behavior

The screen reader would say "Start mode", "Default radio button, 1 of 5" or similar.

### Actual Behavior

Screen reader says "Default radio button, 1 of 5", but not what this is actually referring to. Same with other sets of radio buttons.

The prompts for these groups of radio buttons is mere static text that is not associated to the radio buttons in any way, and the radio buttons aren't grouped in a true groupbox, at least what accessibility is concerned.

**Comment by MarcoZehe** [comment]
This bug is also present in the XAML Toolkit Gallery app: https://github.com/microsoft/Xaml-Controls-Gallery/issues/657

**Comment by chingucoding** [comment]
This is because the WinUI RadioButtons control is actually not accessible: https://github.com/microsoft/microsoft-ui-xaml/issues/3183

**Comment by zadjii-msft** [comment]
For business reasons, I we need to track this in /dup #11248, even though you were definitely first. Sorry about that!
